packages/components/src/components/files/models.ts

```diff
@@ -13,7 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Map } from 'immutable';
+import {List, Map, Record} from 'immutable';
+import {WebDavFile} from "./WebDav";
+import {DomainField} from "../..";
+import {DomainException, DomainIndex} from "../domainproperties/models";
```

Comment:
All of these added imports are unused. They should be removed.

Reply:
Removed